CoreScriptsRoot/Modules/PlayerDropDown.lua

```diff
@@ -190,8 +192,33 @@ end
 
 spawn(function()
 	BlockedList = GetBlockedPlayersAsync()
+	GetBlockedPlayersCompleted = true
 end)
 
+local function getBlockedUserIdsFromBlockedList()
+	local userIdList = {}
+	for userId, _ in pairs(BlockedList) do
+		table.insert(userIdList, userId)
```

Comment:
Could be optimized by changing to
`local userIdList = {}`
`local insert = table.insert`
`for userId, _ in pairs(BlockedList) do`
`insert(userIdList, userId)`